src/Symfony/Bridge/Twig/Extension/SecurityExtension.php

```diff
@@ -21,16 +22,21 @@
  */
 class SecurityExtension extends \Twig_Extension
 {
-    private $context;
+    private $securityChecker;
 
-    public function __construct(SecurityContextInterface $context = null)
+    /**
+     * @param SecurityContextInterface|AuthorizationCheckerInterface
+     *
+     * Passing a SecurityContextInterface as a first argument was deprecated in 2.7 and will be removed in 3.0
+     */
+    public function __construct($securityChecker = null)
```

Comment:
We could typehint AuthorizationCheckerInterface and bump the symfony/security-core requirement